client/components/dashboard/dashboard-service.js

```diff
@@ -79,7 +79,7 @@ const WidgetType = explorer.models.WidgetType;
 explorer.components.dashboard.DashboardService = function(arrayUtilService,
     errorService, widgetFactoryService, dashboardDataService,
     queryBuilderService,  dashboardVersionService, configService,
-    $filter, $location, $rootScope, $timeout, $window) {
+    explorerStateService, $filter, $location, $rootScope, $timeout, $window) {
```

Comment:
I'll add it to the docstring, but the names can't really be reduced as they're used for AngularJS [dependency injection](https://docs.angularjs.org/guide/di); each is the name of an available Angular Service, and AFAIK you can't use an encapsulated object (nor would it necessarily be overall helpful).